.github/CODEOWNERS

```diff
@@ -2,5 +2,6 @@
 *                 @GoogleCloudPlatform/service-extensions-samples-owners @GoogleCloudPlatform/dee-infra
 
 # Sample Reviewers
-samples/          @GoogleCloudPlatform/service-extensions-samples-reviewers @upeeth
+callouts/         @GoogleCloudPlatform/service-extensions-samples-reviewers @upeeth
```

Comment:
Consider adding a leading '/' to `callouts/`, `plugins/`, and `README.md`, so that they match specifically at top-level.

Reply:
Done, good call!